src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Tests/UnitTests/StatisticsServiceTests.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using AutoMapper;
+using DevAdventCalendarCompetition.Repository.Context;
+using DevAdventCalendarCompetition.Repository.Models;
+using DevAdventCalendarCompetition.Services;
+using DevAdventCalendarCompetition.Services.Interfaces;
+using FluentAssertions;
+using Moq;
+using Xunit;
+
+namespace DevAdventCalendarCompetition.Tests.UnitTests
+{
+    public class StatisticsServiceTests
+    {
+        private readonly StatisticsService _controller;
+        private readonly Mock<ITestStatisticsService> _testStatisticsMock;
+
+        // might me useless
+        private readonly Mock<UserTestCorrectAnswer> _testUserTestCorrectAnswerMock;
+
+        public StatisticsServiceTests()
+        {
+            var iDontBelongHere = new UserTestCorrectAnswer { Id = 1, UserId = "1", TestId = 1, AnsweringTime = DateTime.Now, AnsweringTimeOffset = default };
+
+            this._testStatisticsMock = new Mock<ITestStatisticsService>();
+
+            // might be useless
+            this._testUserTestCorrectAnswerMock = new Mock<UserTestCorrectAnswer>();
+
+            this._controller = new StatisticsService(this._testStatisticsMock.Object);
+        }
+
+        [Fact]
+        public void TemporaryTest()
+        {
+            int a = this._controller.TmpStatisticsImpementation("1", 2);
+
+            Assert.True(a == 3, "Temoporary test - wrong!");
```

Comment:
please use 3A rule :)

Reply:
I will use A3 rule when tests will be meaningfull :)